### ISSUE TITLE
Installers

### ISSUE THREAD

**Issue by jesusprubio** [jesusprubio/up#33]
Package managers:
- [ ] yum
- [ ] apt
- [ ] brew
- [ ] winget

**Comment by jesusprubio** [comment]
Yes. We have `goreleaser` now. I haven't tested this but it seems this is what we need: https://goreleaser.com/customization/nfpm/

**Comment by jesusprubio** [comment]
Done here: https://github.com/jesusprubio/up/pull/74

**Comment by jetexe** [comment]
Hay! What about AUR? Since you using `goreleaser` that should not be hard to implement
